samples/subsys/smf/psicc2/src/psicc2_thread.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Glenn Andrews
+ * State Machine example copyright (c) Miro Samek
+ *
+ * Implementation of the statechart in Figure 2.11 of
+ * Practical UML Statecharts in C/C++, 2nd Edition by Miro Samek
+ * https://www.state-machine.com/psicc2
+ * Used with permission of the author.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#define PSICC2_THREAD_STACK_SIZE       1024
+#define PSICC2_THREAD_PRIORITY         7
+#define PSICC2_THREAD_EVENT_QUEUE_SIZE 10
+
+/**
+ * @brief Event to be sent to an event queue
+ */
+struct psicc2_event {
+	uint32_t event_id;
+} __aligned(4);
```

Comment:
The alignment of this struct is already that of `event_id` (4 bytes). There is no need for explicit alignment.
```suggestion
};
```